OWNERS

```diff
@@ -0,0 +1,16 @@
+approvers:
+- heyitsanthony
+- philips
+- fanminshi
+- gyuho
+- mitake
+- jpbetz
+- xiang90
+reviewers:
```

Comment:
we can add more people to reviewers though.

Reply:
Just added more contributors from Google, Red Hat, and CockroachDB.